src/k2/cmd/demo/tso_sample_app.cpp

```diff
@@ -141,7 +141,7 @@ int main(int argc, char **argv)
     app.addOptions()
     ("tso_endpoint", bpo::value<k2::String>()->default_value("tcp+k2rpc://127.0.0.1:9000"), "URL of Timestamp Oracle (TSO), e.g. 'tcp+k2rpc://192.168.1.2:12345'");
 
-    app.addApplet<k2::TSO_ClientLib>(1s);
+    app.addApplet<k2::TSO_ClientLib>();

```

Comment:
I don't think this problem is fixed is it? Here the problem is the client tried to connect the server before the server was ready, but this fix is for if the user of the client tries to get a timestamp before the client is ready.

Reply:
The true bug is that client request should not go through and wait until the server discover is done, which is this change about. This delay start hack was used to avoid one of two reasons this case could happen, which is that the server is not started yet. It is no longer needed here that we have delayed retry to connect to server in this tsoclient, which should be sufficient time wise in normal case, including in test.
So this hack should not be needed with the fix here.